console/console-init/ui/src/modules/iot-detail/components/IoTProjectDetailHeader/IoTProjectDetailHeader.tsx

```diff
@@ -109,31 +108,18 @@ const IoTProjectDetailHeader: React.FunctionComponent<IIoTProjectDetailHeaderPro
       </SplitItem>
     </>
   );
-
-  const EnabledIcon = () => {
-    return isEnabled ? (
-      <span>
-        <ToggleOnIcon
-          onClick={() => {
-            changeEnable(false);
-          }}
-          color="var(--pf-global--active-color--100)"
-          size={IconSize.lg}
-        />
-        &nbsp; Enabled
-      </span>
-    ) : (
-      <span>
-        <ToggleOffIcon
-          onClick={() => {
-            changeEnable(true);
-          }}
-          size={IconSize.lg}
-        />
-        &nbsp; Disabled
-      </span>
-    );
+  const onChange = () => {
+    changeEnable(!isEnabled);
   };
+  const EnabledIcon = (
```

Comment:
is there any specific reason to use as a variable, why not function.
can convert it into function and call it accordingdly.

  const EnabledIcon =()=>(
    <Switch
      id="simple-switch"
      label="View in Json"
      labelOff="View in Json"
      isChecked={isEnabled}
      onChange={onChange}
    />
  );

//call it as component
<EnabledIcon/>

Reply:
I replaced this with Switch component a PR.